app/src/main/java/com/example/androidenhancedvideoplayer/MainActivity.kt

```diff
@@ -47,71 +47,36 @@ class MainActivity : ComponentActivity() {
 }
 
 @Composable
-fun VideoFromURL() {
+fun VideoFromURL(url: String = ExampleUrl.MP4) {
     EnhancedVideoPlayer(
-        mediaItem = MediaItem.fromUri(ExampleUrl.MP4),
+        mediaItem = MediaItem.fromUri(url),
         zoomToFit = false,
         enableImmersiveMode = true,
         alwaysRepeat = false,
         settingsControlsCustomization = SettingsControlsCustomization(
             speeds = listOf(0.5f, 1f, 2f, 4f)
-        )
+        ),
+        soundOff = false
     )
 }
 
 @Composable
 fun VideoFromHLSUri() {
-    EnhancedVideoPlayer(
-        mediaItem = MediaItem.fromUri(ExampleUrl.HLS),
-        zoomToFit = false,
-        enableImmersiveMode = true,
-        alwaysRepeat = true,
-        settingsControlsCustomization = SettingsControlsCustomization(
-            speeds = listOf(0.5f, 1f, 2f, 4f)
-        )
-    )
+    VideoFromURL(ExampleUrl.HLS)
 }
 
 @Composable
 fun VideoFromDASHUri() {
-    EnhancedVideoPlayer(
-        mediaItem = MediaItem.fromUri(ExampleUrl.DASH),
-        zoomToFit = false,
-        enableImmersiveMode = true,
-        alwaysRepeat = false,
-        settingsControlsCustomization = SettingsControlsCustomization(
-            speeds = listOf(0.5f, 1f, 2f, 4f)
-        )
-    )
+    VideoFromURL(ExampleUrl.DASH)
 }
 
 @Composable
 fun VideoFromMSSUri() {
-    EnhancedVideoPlayer(
-        mediaItem = MediaItem.fromUri(ExampleUrl.MSS),
-        zoomToFit = false,
-        enableImmersiveMode = true,
-        alwaysRepeat = false,
-        settingsControlsCustomization = SettingsControlsCustomization(
-            speeds = listOf(0.5f, 1f, 2f, 4f)
-        )
-    )
+    VideoFromURL(ExampleUrl.MSS)
 }
 
 @Composable
 fun VideoFromResources() {
     val context = LocalContext.current
-
-    EnhancedVideoPlayer(
-        mediaItem = MediaItem.fromUri(
-            "android.resource://${context.packageName}/raw/spinning_earth"
-        ),
-        zoomToFit = false,
-        enableImmersiveMode = true,
-        disableControls = false,
-        alwaysRepeat = false,
-        settingsControlsCustomization = SettingsControlsCustomization(
-            speeds = listOf(0.5f, 1f, 2f, 4f)
-        )
-    )
+    VideoFromURL("android.resource://${context.packageName}/raw/spinning_earth")
 }
```

Comment:
There is no need to keep all these VideoFrom composable, IMO we should remove them all and use the EnhancedVideoPlayer with the URL directly on the onCreate method. For the resource video, I wouldn't care if we set the raw path manually as this is a sample app. What do you think?

Example:
```kotlin
object ExampleUrl {
    // ....
    val RESOURCE = "android.resource://com.example.androidenhancedvideoplayer/" +
        R.raw.spinning_earth
}
```


```kotlin
override fun onCreate(savedInstanceState: Bundle?) {
        WindowCompat.setDecorFitsSystemWindows(window, false)
        super.onCreate(savedInstanceState)
        setContent {
            AndroidEnhancedVideoPlayerTheme {
                val orientation = LocalConfiguration.current.orientation
                Column(
                    modifier = Modifier
                        .fillMaxWidth()
                        .safeDrawingPadding()
                ) {
                    Box(
                        modifier = Modifier
                            .fillMaxSizeOnLandscape(orientation = orientation)
                            .fillMaxWidth()
                    ) {
                        EnhancedVideoPlayer(
                            mediaItem = MediaItem.fromUri(url),
                                zoomToFit = false,
                                enableImmersiveMode = true,
                                alwaysRepeat = false,
                                settingsControlsCustomization = SettingsControlsCustomization(
                                    speeds = listOf(0.5f, 1f, 2f, 4f)
                                ),
                                soundOff = false
                            )
                    }
                    RecommendedVideosComponent()
                }
            }
        }
    }
}
```